apps/discovery-platform/src/index.ts

```
@@ -84,9 +99,37 @@
     if (!process.env.URL) {
         throw new Error("Missing 'URL' env var.");
     }
-    // postgres url
-    if (!process.env.DATABASE_URL) {
-        throw new Error("Missing 'DATABASE_URL' env var.");
+    // postgres host
+    if (!process.env.PGHOST) {
```

ESLint

[error] PGHOST is not listed as a dependency in turbo.json

(turbo/no-undeclared-env-vars)